__tests__/newrelic.tests.js

```diff
@@ -1,14 +1,21 @@
 const newrelic = require( '../src/newrelic/' );
 
 describe( 'src/newrelic', () => {
-	describe( 'Should fail if some environment variables are not present', () => {
-		const OLD_ENV_VARS = process.env;
+	const OLD_ENV_VARS = process.env;
 
-		// eslint-disable-next-line no-undef
-		afterEach( () => {
-			process.env = OLD_ENV_VARS;
-		} );
+	beforeEach( () => {
+		jest.resetModules();
+
+		jest.doMock( 'newrelic', () => ( {
```

Comment:
We can just use `jest.mock` here; `doMock` is only needed if using `babel-jest`.